test/puppeteer.spec.js

```diff
@@ -255,6 +255,101 @@ module.exports.addTests = function(runner, expect, defaultBrowserOptions, puppet
         expect(fs.existsSync(executablePath)).toBe(true);
       });
     });
+    describe('DeterministicMode', function() {
+      it('should let us override the date', async({server}) => {
+        const browser = await puppeteer.launch({deterministic: {date: new Date('Jan 1, 1980')} });
```

Comment:
@ADClarke you should re-use `defaultBrowserOptions` so that this launches well on Ci's